index.html

```diff
@@ -0,0 +1,104 @@
+    
+    <p>You can find more information about <a href="https://upel.edu.ve/"> important university dates</a> on our website.</p>
+    
+    
+    <h2>Subjects of study</h2>
+    
+    <p>At the Awesome Science Faculty, we have a pretty open-minded research facility — as long as the subjects fall somewhere in the realm of science and technology. You seem like an intelligent, dedicated researcher, and just the kind of person we'd like to have on our team. Saying that, of the ideas you submitted we were most intrigued by are as follows, in order of priority:</p>
+    <ol>
+      <li> Turning H<sub>2</sub>O into wine, and the health benefits of Resveratrol (C <sub>14</sub>H<sub>12</sub>O<sub>3</sub>)</li>
+      <li> Measuring the effect on performance of funk bassplayers at temperatures exceeding 30&deg;C (86&#176;F), when the audience size exponentially increases (effect of 3 &#215; 10 <sup>3</sup> increasing to 3 &times; 10 <sup>4</sup>.)</li>
+      <li> <abbr title="HyperText Markup Language">HTML</abbr>  and <abbr title="Cascading Style Sheets">CSS</abbr> constructs for representing musical scores.</li>
+    </ol>
+    <p>So please can you provide more information on each of these subjects, including how long you'd expect the research to take, required staff and other resources, and anything else you think we'd need to know? Thanks.</p>
+    
+    
+    <h2><em>Exotic dance moves</em></h2>
+    
+    <p>Yes, you are right! As part of my post-doctorate work, I did study exotic tribal dances. To answer your question, my favourite dances are as follows, with definitions:</p>
+    <dl>
+      <dt>Polynesian chicken dance</dt>
+      <dd><sub> A little known but very influential dance dating back as far as 300BC, a whole village would dance around in a circle like chickens, to encourage their livestock to be "fruitful".</dd>
+      <dt>Icelandic brownian shuffle</dt>
+      <dd>Before the Icelanders developed fire as a means of getting warm, they used to practice this dance, which involved huddling close together in a circle on the floor, and shuffling their bodies around in imperceptibly tiny, very rapid movements. One of my fellow students used to say that he thought this dance inspired modern styles such as Twerking.</dd>
+      <dt>Arctic robot dance</dt>
+      <dd>An interesting example of historic misinformation, English explorers in the 1960s believed to have discovered a new dance style characterized by "robotic", stilted movements, being practiced by inhabitants of Northern Alaska and Canada. Later on however it was discovered that they were just moving like this because they were really cold.</dd>
+    </dl>
+    <p>For more of my research, see my <a href="http://example.com.">exotic dance research page.</a> </p>
+    
+    <em> Yours sincerely,</em> <br>
+    <br>
+    <br>
+    <br>
+    <br>
+    Dr Eleanor Gaye <br>
+    <br>
+    University of Awesome motto: <q>Be awesome to each other.</q>  -- The memoirs of Bill S Preston, Esq
```

Comment:
` -- <cite>The memoirs of Bill S Preston, <abbr
        title="Esquire">Esq.</abbr></cite></p>`